### DISCUSSION TITLE
I am a rookie. According to the tutorial, I learned the last step and encountered the following problem

### DISCUSSION THREAD

**Discussion by YYongzhao11** [Kitware/trame#68] [Q&A] [answered]
return server.start_webserver(options=options, protocol=CoreServer, **kwargs)
TypeError: start_webserver() got an unexpected keyword argument 'exec_mode'
How to modify it?
![QQ截图20220520160847](https://user-images.githubusercontent.com/105912017/169483918-cf91e694-ef82-46ec-af13-c6891d234994.jpg)

**Comment by jourdain** [comment]
Can you explain what you did to install trame?

**Comment by YYongzhao11** [comment]
I use the trace installed by PIP according to the tutorial you gave me. . From "setup for vtk" to "application" are correct. There was an error in the last part "paraview". There may be an error when setting the paraview environment in the first step. I don't know how to set it. "ParaView comes with its own Python, which may be missing some dependencies for the desired usage. We can add more Python packages into ParaView by create a virtual environment and activate it inside your application using that import line from paraview.web import venv or by using our local version and importing it." I didn't understand the step. I set the value "PYTHONPATH" of the environment variable to the address of "D:\Program Files\ParaView 5.10.0\bin\Lib\site-packages".


On 5/20/2022 22:35，Sebastien ***@***.***> wrote：

Can you explain what you did to install trame?

**Comment by jourdain** [comment]
trame v2 will only work with ParaView 5.11 and on top of that you can not have a **vtk** package in your venv as it will conflict with the one inside `pvpython`.

The error you are getting here is related to the fact that PV 5.10 is bundling a version of wslink that is not compatible with trame v2.

**Comment by YYongzhao11** [comment, marked as answer]
how to download pv5.11



---- 回复的原邮件 ----
| 发件人 | Sebastien ***@***.***> |
| 日期 | 2022年05月21日 00:17 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [Kitware/trame] I am a rookie. According to the tutorial, I learned the last step and encountered the following problem (Discussion #68) |

trame v2 will only work with ParaView 5.11 and on top of that you can not have a vtk package in your venv as it will conflict with the one inside pvpython.

The error you are getting here is related to the fact that PV 5.10 is bundling a version of wslink that is not compatible with trame v2.

**Reply by jourdain** [reply]
Until it is released, you will need to get the [Nightly version](https://www.paraview.org/download/?version=nightly)
